src/app/themes/theme.js

```diff
@@ -201,6 +201,47 @@ const theme = createMuiTheme({
         width: '100%',
       },
     },
+
+    MuiTablePagination:{
+      root:{
+        display: 'flex',
+        alignItems: 'center',
+        alignContent: 'center',
+        justifyContent: 'center',
+        flexWrap: 'wrap',
+        fontSize:'1em',
+      },
+      toolbar:{
+        backgroundColor:'#ffffff',
+        margin:'16px 0px 4px 0px',
+        boxShadow:'0px 2px 1px -1px rgb(0 0 0 / 20%), 0px 1px 1px 0px rgb(0 0 0 / 14%), 0px 1px 3px 0px rgb(0 0 0 / 12%)',
```

Comment:
This looks like a super complicated box shadow. What does it do?

Reply:
This box shadow was taken from Material UI box shadow library. I wanted to use a "set" box shadow for our pagination toolbar. The elevation prop did not work on this element.

Reply:
Where was it taken from? I'm asking because I am trying to interpret these numbers to see what it does, and from what I can tell it looks like it's stacking three nearly transparent blank 1 pixel tall lines on top of each other below the element the shadow is coming from. I put it into a box-shadow playground to see what it looked like, and had to zoom my monitor by 175% to actually see anything happen when toggling the shadow on and off

Reply:
If you go to https://material-ui.com/system/shadows/ and inspect the box shadows from box 1 you'll see where I pulled this shadow from.